### ISSUE TITLE
Android 13 - Not support (API 33 + Android 12.0.1) -  Black Screen.

### ISSUE THREAD

**Issue by michelsondan** [DmcSDK/cordova-plugin-mediaPicker#130]
Hello,

After upgrading to cordova android 12.0.1  + API level 33
On android device with version 13 - users are getting black screen.

Any solution?

Thank you.

**Comment by itbeyond** [comment]
I confirm this is a problem under API 33 and cordova android 12.0.1. API's below 33 work as expected.

**Comment by NeilCresswell** [comment]
As Google have a requirement to target SDK 33, we had to switch to a different image picker and ended up with `cordova plugin add github:Kersei/cordova-plugin-advanced-imagepicker`, which works fine for us using the v12.0.1 Android platform and targetting SDK v33. Tested on real Android 11, 12 and 13 devices, as well as on the latest iOS since the picker also supports that.

This alternative picker is a fork of another picker that has the same issue with targeting API 33 so someone kindly coded a fix, and we simply couldn't find another working solution. It isn't widely adopted, and the underlying picker it uses does comes with it's own quirks;-

**iOS**: No bugs but always auto-plays the first video in the list or any video selected, which may be annoying. It does have another huge advantage for us though in that we can finally specify to get the original resolution of the video, something that cordova-plugin-mediaPicker always reduced on iOS. I believe it also handles HEIC photo conversion to JPEG nicely too.

**Android (All Versions)**: Ignores any min or max quantity and always allows a pick of up to 10 videos or photos. However, at least videos don't auto-play.

**Android < 13**: The first thumbnail is actually a link to record/snap a video or photo, so you need to request those permissions too else selecting the first thumbnail will cause the plugin to terminate. We don't get this issue on Android 13.

We're extremely grateful to have something that works so we could make the Google deadline for changing to target API v33, so we found these quirks to be acceptable. Hope this helps.

**Comment by Mr-Anonymous** [comment]
I have the same issue. In Android 33 it shows blank screen when selecting an image. I am going to try the `cordova-plugin-advanced-imagepicker` as suggested by @NeilCresswell  since this plugin here seems to be abandoned unfortunately.

**Comment by Mr-Anonymous** [comment]
I tried installing `Kersei/cordova-plugin-advanced-imagepicker` and it doesn't work for Android 13 either. Some suggested to change to `"ANDROID_IMAGE_PICKER_VERSION": "1.3.2"` but that doesn't work for me too. I use Android 9 ionic 1 app and I cant update to cordova android 12 yet. So it seems all plugins are reaching a deadend with Android 13 support. Really stuck here since Google is forcing us to target sdk 33 from now on. Any other solutions please?

**Comment by itbeyond** [comment]
The UI is not as nice as it is portrait only but I went with cordova-plugin-advanced-imagepicker 1.6.2 and set ANDROID_IMAGE_PICKER_VERSION to 1.4.2 in config.xml (<preference default="1.4.2" name="ANDROID_IMAGE_PICKER_VERSION" />)

**Comment by Mr-Anonymous** [comment]
Thank you @itbeyond I tried that, both 1.4.2 and 1.3.2 version for `ANDROID_IMAGE_PICKER_VERSION` in that plugin. I also tried updating this value to 1.4.2 in the plugin.xml but I still have the same issue in Android 13 where it doesn't respond. I use cordova android@9.0.0. I read somewhere that this seems to work for cordova android@12.0.0 so I wonder if that is the issue for me. I am unable to upgrade my project to android@12.0.0 since that gives a whole lot more trouble since it is a very old ionic 1 based project. Also tried other image picker plugins and all have the issues with Android 13.
